examples/32-create-public-tag.js

```diff
@@ -0,0 +1,5 @@
+module.exports = function (migration) {
+  migration.createTag('publicsampletag', {
+    name: 'public-marketing'
+  }, 'public');
```

Comment:
Did you do it this way because the `visibility` is not changeable later so we don't want it as a separate action?

Reply:
Can't speak for Marco but that makes sense.